### PR TITLE
Sendkit Mail stream Endpoint Documentation

### PR DESCRIPTION
**Pull request by ejpersson**
Initial documentation of the mail stream endpoint in sendkit available in the Suite REST API.

### CODE REVIEW

**Comment by ejpersson** [comment]
@boogie please have a look at this documentation for the mailstream endpoint. I've tried to keep it as close to the rest of the suite documentation as possible.

File: source/suite/mailstreams/index.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Mail Streams
+============
+
+In Sendkit, mails are being sent asynchronously over so-called _mail streams_, referenced by their ID.
```
Comment (emkulovits) [inline source/suite/mailstreams/index.rst]: will it be called "sendkit"? atm it's "Real Time Mailing"...

**Comment by emkulovits** [comment]
@boogie is there any way to see the final docs as they'd look like? e.g. on trunk-int?

File: source/suite/mailstreams/index.rst (hunk 1 above)
Comment (ejpersson) [inline source/suite/mailstreams/index.rst]: Since this is not yet decided I left it like it was.

**Comment by boogie** [comment]
@ejpersson Please note that this is a public repository. 
@emkulovits You can generate these documents locally. Just "easy_install sphinx", and then "make build html", and you can open the docs from the build/html folder.

I'll review this documentation, but give me some time to do it.

**Comment by ejpersson** [comment]
@boogie do you have another workflow for documentation changes than pull requests? Since it's public documentation anyway I don't see the harm.

**Comment by ejpersson** [comment]
@boogie btw, how do you usually deal with documentation for not yet fully available features. We have some piloting clients that would need access to the documentation. Do you have some staging documentation somewhere?

**Comment by boogie** [comment]
If it's not linked from other pages, you can push it to the main branch. And if it's ready, we start linking it. We have no staging site at the moment.

**Comment by ejpersson** [comment]
@boogie excellent, I've removed the any references from the navigation and list of end points

**Comment by ejpersson** [comment]
@boogie so I can merge the pull request?

**Comment by ejpersson** [comment]
is it then automatically build and made available on dev.emarsys.com?

**Comment by boogie** [comment]
Yep.

**Comment by boogie** [comment]
Yes: https://dev.emarsys.com/suite/mailstreams/index.html

**Pull request merged**